administrator/language/de-DE/com_users.ini

```
@@ -400,6 +400,10 @@ COM_USERS_USER_BACKUPCODES_CAPTIVE_PROMPT="Falls die Anmeldung mit der gewohnten
 COM_USERS_USER_BACKUPCODES_DESC="Ermöglicht den Zugriff auf die Website, wenn alle anderen eingerichteten Methoden der Multi-Faktor-Authentifizierung fehlschlagen."
 COM_USERS_USER_BATCH_FAILED="Ein Fehler trat auf beim Ausführen der Stapelverarbeitung: %s."
 COM_USERS_USER_BATCH_SUCCESS="Stapelverarbeitung ausgeführt."
+COM_USERS_USER_COLORSCHEME_LABEL="Dark Mode"
+COM_USERS_USER_COLORSCHEME_OPTION_DARK="Dunkles Farbschema verwenden"
+COM_USERS_USER_COLORSCHEME_OPTION_FOLLOW_OS="Betriebssystem-Einstellungen beachten"
```

```suggestion
COM_USERS_USER_COLORSCHEME_OPTION_FOLLOW_OS="Betriebssystem-Einstellungen verwenden"
```

Würde beidesmal 'verwenden' nehmen oder das Wort ganz weglassen